.github/workflows/integration-tests.yml

```diff
@@ -45,4 +45,4 @@ jobs:
       - name: run tests
         run: make testall
         env:
-          LINODE_API_TOKEN: ${{ secrets.DX_LINODE_TOKEN }}
+          LINODE_API_TOKEN: ${{ secrets.DX_LINODE_TOKEN }}
```

Comment:
Can we have a blank line at the end of the file to fix the GitHub warning?
<img width="90" alt="image" src="https://github.com/linode/ansible_linode/assets/121905282/baa6b7a6-1899-43fb-a580-f39944f114da">